packages/hardhat-core/sample-projects/advanced/.env.template

```diff
@@ -0,0 +1,3 @@
+ETHERSCAN_API_KEY=
+ROPSTEN_URL=
+PRIVATE_KEY=
```

Comment:
```suggestion
ETHERSCAN_API_KEY="ABC123ABC123ABC123ABC123ABC123ABC1"
ROPSTEN_URL="https://eth-ropsten.alchemyapi.io/v2/<YOUR ALCHEMY KEY>"
PRIVATE_KEY="0xabc123abc123abc123abc123abc123abc123abc123abc123abc123abc123abc1"
```

Reply:
Addressed in https://github.com/nomiclabs/hardhat/pull/1632/commits/aaac3031ec9416a5a467d81e7e36386f0481687e